src/main/java/eu/miaplatform/customplugin/springboot/GracefulShutdown.java

```diff
@@ -0,0 +1,22 @@
+package eu.miaplatform.customplugin.springboot;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationListener;
+import org.springframework.context.event.ContextClosedEvent;
+
+public class GracefulShutdown implements ApplicationListener<ContextClosedEvent> {
+
+    private static final Logger logger = LoggerFactory.getLogger(GracefulShutdown.class);
+
+    @Override
+    public void onApplicationEvent(ContextClosedEvent contextClosedEvent) {
+        try {
+            logger.info("START GracefulShutdown...");
```

Comment:
Please align these logs with the other logs in the library, if the library presents no log at all remove them.